.gitignore

```diff
@@ -0,0 +1,2 @@
+modules/
+bolt.yaml
```

Comment:
Why bolt.yaml?

Reply:
I had one locally for testing :shrug: - I don't expect anyone will want to commit a bolt config here